vignettes/predict.Rmd

```diff
@@ -0,0 +1,109 @@
+---
+title: "Predicting from a MMRM"
+package: mmrm
+output:
+  rmarkdown::html_document:
+          theme: "spacelab"
+          highlight: "kate"
+          toc: true
+          toc_float: true
+vignette: |
+  %\VignetteIndexEntry{Predicting from a MMRM}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options:
+  chunk_output_type: console
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+library(mmrm)
+```
+
+## Prediction of conditional mean
+
+Let $Y^{(i)}$ be the $n$ outcomes of the $i$-th subject and $X^{(i)}$ the $n\times p$
```

Comment:
sorry for being picky but it would be important to keep the same notation as in the other vignettes, you can take the model fitting algorithm one as the reference

Reply:
this will help the readers of the documentation, and also us once we decide to write a JSS paper sourcing material from the vignettes :-)

Reply:
hm, found that a bit clunky for this purpose - we can align those later once we know how to implement this - for now the vignette is the design doc / scratchbook

Reply:
updated, should be more aligned now